src/pkg/account/account.go

```diff
@@ -88,3 +91,12 @@ func (a Account) ID() string {
 
 	return a.Config[a.Provider.String()+"-tenant-id"]
 }
+
+func (a Account) GetAccountConfigForHash() (map[string]any, error) {
+	switch a.Provider {
+	case ProviderM365:
+		return a.GetM365ConfigForHashing()
+	}
+
+	return nil, clues.New(fmt.Sprintf(invalidProviderErrTemplate, a.Provider.String()))
```

Comment:
Per error and logging conventions, lets keep templates out of the error message, please.  This can be handled as:
```suggestion
	return nil, clues.Stack(errInvalidProvider).With("provider", a.Provider)
```